core/src/test/java/io/snyk/plugins/artifactory/scanner/PythonScannerTest.java

```diff
@@ -41,7 +41,7 @@ void shouldTestPipPackage() throws Exception {
     TestResult result = scanner.scan(fileLayoutInfo, repoPath);
     assertFalse(result.success);
     assertEquals(1, result.dependencyCount);
-    assertEquals(5, result.issues.vulnerabilities.size());
+    assertEquals(6, result.issues.vulnerabilities.size());
```

Comment:
Had to bump this to 6 as this seems to use real data and number of vulns has increased.